docs/Getting Started/Installation/index.md

```diff
@@ -189,3 +189,21 @@ from within the JavaEE directory and a JAR should be generated in the build/libs
 Glassfish 5.0.0 only works on JDK 8 and lower.
 !!!
 !@
+
+@![javascript]
+The SDL JavaScript Suite comes with prebuilt bundles of the library. There is a vanilla JavaScript distribution of the library as well as one for Node.js. They are located in the lib/js/dist and lib/node/dist directories respectively. To use one of these bundles, include one of the following somewhere in your project:
+
+        const SDL = require('lib/js/SDL.min.js');
+OR
+
+        const SDL = require('lib/node/SDL.min.js');
+
+### Building The JavaScript Library    
+To build the library bundle from the source code, first clone the [SDL JavaScript Suite](https://github.com/smartdevicelink/sdl_javascript_suite) repository then, simply call:
```

Comment:
A developer will need to set up git submodules so that the generator can reference the rpc_spec. Recommend trying to do a fresh clone of the JS suite and double checking that the build steps here will work